CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.4.1] - 2020-09-18
```

Comment:
Remove it entirely and leave your changed in the the unreleased section as I want to put other things into 0.4.1.